.github/workflows/test.yml

```
@@ -0,0 +1,232 @@
+          mamba activate test
+          pip install -U ./partcad ./partcad-cli
+          mamba deactivate
+
+  test-examples-partcad:
+    strategy: *strategy
+    runs-on: ${{ matrix.os }}
+    if: "!startsWith(github.event.head_commit.message, 'Version updated')"
+    needs: test-cli-setup
+    timeout-minutes: 10
+
+      - name: Basic integration test for CLI (PartCAD examples)
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 150: mapping values are not allowed in this context

(syntax-check)